_profile/prateek.md

```diff
@@ -12,13 +12,13 @@ name: Prateek Tiwari
 # Optional, atleast 2 recommended
 email: nopanictri@yahoo.com
 quora: https://www.quora.com/profile/Prateek-Tiwari-62
-google:https://plus.google.com/109674544460125183017
+google: https://plus.google.com/109674544460125183017
 github: https://github.com/pitbuk101 
 youtube: https://www.youtube.com/channel/UCCVT_NxzDnb2pMvTCSw5A1A
 instagram: https://www.instagram.com/prateek_tiwari_99/
 
 # Optional
 # Upload the image to imgur and
-# paste the direct link to image
+# paste the direct link to imag
 picture: https://imgur.com/a/EdaOCpp
```

Comment:
Prateek, please update the image link too. Check the related comment to find the link you should be using.